apps/authentication-portal/src/main/webapp/device-success.jsp

```diff
@@ -0,0 +1,186 @@
+<!doctype html>
+<html>
+
+<head>
+    <!-- header -->
+    
+    
+    
+    <!-- localize.jsp MUST already be included in the calling script -->
+    
+    
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link rel="icon" href="libs/theme/assets/images/favicon.ico" type="image/x-icon" />
+    <link href="libs/theme/wso2-default.min.css" rel="stylesheet">
+    
+    <title>WSO2 Identity Server</title>
+    
+    <style>
+        html,
+        body {
+            height: 100%;
+        }
+        
+        body {
+            flex-direction: column;
+            display: flex;
+        }
+        
+        main {
+            flex-shrink: 0;
+        }
+        
+        main.center-segment {
+            margin: auto;
+            display: flex;
+            align-items: center;
+        }
+        
+        main.center-segment>.ui.container.medium {
+            max-width: 450px !important;
+        }
+        
+        main.center-segment>.ui.container.large {
+            max-width: 700px !important;
+        }
+        
+        main.center-segment>.ui.container>.ui.segment {
+            padding: 2.5em 1.6rem;
+        }
+        
+        main.center-segment>.ui.container>.ui.segment .segment-form .buttons {
+            margin-top: 1em;
+        }
+        
+        main.center-segment>.ui.container>.ui.segment .segment-form {
+            text-align: left;
+        }
+        
+        main.center-segment>.ui.container>.ui.segment .segment-form .align-center {
+            text-align: center;
+        }
+        
+        main.center-segment>.ui.container>.ui.segment .segment-form .align-right {
+            text-align: right;
+        }
+        
+        main #toc {
+            position: sticky;
+            top: 93px;
+        }
+        
+        main #toc>nav.table-of-contents ol>li:not(.sub) {
+            list-style: square;
+        }
+        
+        main #toc>nav.table-of-contents ol>li:not(.sub):before {
+            display: none;
+        }
+        
+        main #toc>nav.table-of-contents ol>li.sub {
+            margin-left: 20px;
+        }
+        
+        footer {
+            padding: 2rem 0;
+        }
+        
+        body .product-title .product-title-text {
+            margin-top: 0;
+        }
+    </style>
+
+
+
+
+</head>
+
+<style>
+    
+    #Para1{
```

Comment:
Use better id name. And try to use existing styles from the theme or if there is not matching one move this to header include file.